lib/nerves_ssh/options.ex

```diff
@@ -17,7 +17,8 @@ defmodule NervesSSH.Options do
 
   alias Nerves.Runtime.KV
 
-  @default_system_dir "/etc/ssh"
+  @default_host_system_dir "/etc/ssh"
+  @default_device_system_dir "/data/ssh"
```

Comment:
```suggestion
  @default_device_system_dir "/data/nerves_ssh"
```

I've been trying to follow the convention `/data/package_name` for everything new that gets written to `/data`.